Week02/mlang_01.ipynb

```diff
@@ -0,0 +1,188 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "area_linear = simps(linear(x),x)\n",
+    "area_cubic = simps(cubic(x),x)\n",
+    "print(area_linear)\n",
+    "print(area_cubic)\n",
+    "\n",
+    "plt.scatter(x,y)\n",
+    "plt.plot(x,linear(x))\n",
+    "plt.plot(x,cubic(x))\n",
+    "\n",
+    "area1 = r\"$\\int_{x_0}^{x_{n-1}} linear(x)\\mathrm{d}x = $\" + \"{:.0f}\".format(area_linear)\n",
```

Comment:
Wooo LateX!!!!! 🎉 My favorite thing about matplotlib is that I can use LaTeX to render symbols in my plot labels and legends. Nice work!